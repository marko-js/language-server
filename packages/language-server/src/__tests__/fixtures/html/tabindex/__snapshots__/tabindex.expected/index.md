## Diagnostics
### Ln 1, Col 2
```marko
> 1 | <button tabindex=3>My Button</button>
    |  ^^^^^^ Fix any of the following:
  Element has a tabindex greater than 0
```

