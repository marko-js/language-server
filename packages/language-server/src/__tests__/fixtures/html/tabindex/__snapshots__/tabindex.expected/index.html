<button data-marko-node-id="0" tabindex="3">My Button</button>